blocks/file-blocks/excalidraw/index.tsx

```diff
@@ -16,6 +16,7 @@ if (typeof window !== "undefined") {
 export default function (props: FileBlockProps) {
   const { context, content, isEditable, onUpdateContent } = props;
   const [excalModule, setExcalModule] = useState<any>(null);
+  console.log("excalModule", excalModule);
```

Comment:
really wish there was a :log: in PR comments

Reply:
doh!